bin/reth/Cargo.toml

```diff
@@ -67,6 +67,8 @@ reth-consensus.workspace = true
 reth-optimism-primitives.workspace = true
 reth-engine-util.workspace = true
 reth-prune.workspace = true
+reth-stages-api.workspace = true
+reth-optimism-cli.workspace = true
```

Comment:
this is likely problematic because this will enable a ton of op features.

so we need to replicate this solution for the reth-optimism-cli crate for now:

https://github.com/paradigmxyz/reth/blob/405b73045549611c3381020cef9302b4ee3f483c/crates/optimism/evm/Cargo.toml#L39-L45

https://github.com/paradigmxyz/reth/blob/405b73045549611c3381020cef9302b4ee3f483c/crates/optimism/evm/src/lib.rs#L9-L10